packages/plugins/src/cli/deploy.ts

```diff
@@ -277,13 +326,65 @@ export const deploy = async (
     await sphinxContext.prompt(previewString)
   }
 
-  const { receipts } = await runEntireDeploymentProcess(
-    parsedConfig,
-    merkleTree,
+  const treeSigner = {
+    signer: await signer.getAddress(),
+    signature: await signMerkleRoot(merkleTree.root, signer),
+  }
+  const deployment: Deployment = {
+    id: 'only required on website',
+    multichainDeploymentId: 'only required on website',
+    projectId: 'only required on website',
+    chainId: parsedConfig.chainId,
+    status: 'approved',
```

Comment:
Well, from the perspective of the website, `approved` means the user has submitted the signatures required to execute the deployment. `empty` doesn't make any sense in that context either. I sympathize with your viewpoint on this, but IMO, it's not worth the effort to refactor the website just to update this to a value that makes more sense in both situations.

Reply:
I think this is a footgun since 'approved' always means 'approved on-chain' in the monorepo. Imo calling it 'signed' would be safer. It's not worth stalling this PR over it, but just want to point it out